src/views/CardStack/CardStackStyleInterpolator.js

```diff
@@ -51,33 +53,25 @@ function forHorizontal(
   if (!layout.isMeasured) {
     return forInitial(props);
   }
+  const interpolate = getFirstAndLastSceneIndicesForInterpolationInputRange(props); // eslint-disable-line
```

Comment:
This is kinda my mistake, but um... it would be nicer not to use `eslint-disable-line` so many times... I'm guessing it's because of the long function name? If that's the cause, maybe `getSceneIndicesForInterpolationInputRange` will make eslint happy?